utils/tfhe-versionable/examples/transparent_then_not.rs

```diff
@@ -0,0 +1,162 @@
+//! This example is similar to the "transparent" one, except that the wrapper type is transparent at
+//! a point in time, then converted into its own type that is not transparent.
+//!
+//! To make this work, we consider that the inner and the wrapper type share the same history up to
+//! the version where the transparent attribute has been removed.
+
+use std::convert::Infallible;
```

Comment:
Ok I will try to make the example more readable. I don't think this is just an ordering issue because I would like the order of the definitions the match the explanation in the comments